### ISSUE TITLE
Can launch on arch linux

### ISSUE THREAD

**Issue by orblazer** [ATLauncher/ATLauncher#337]
### Description

When i try run the launcher i am lock on splash screen.

### Steps to Reproduce

1. Download last version of ATLauncher.jar
2. Start ATLauncher with `java -jar ATLauncher.jar`

**Expected behavior:** [What you expect to happen]

Start the launcher

**Actual behavior:** [What actually happens]

Error when start :
```
java.lang.reflect.InaccessibleObjectException: Unable to make public long com.sun.management.internal.OperatingSystemImpl.getTotalPhysicalMemorySize() accessible: module jdk.management does not "opens com.sun.management.internal" to unnamed module @53fd0d10
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:340)
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:280)
        at java.base/java.lang.reflect.Method.checkCanSetAccessible(Method.java:198)
        at java.base/java.lang.reflect.Method.setAccessible(Method.java:192)
        at com.atlauncher.utils.Utils.getSystemRam(Utils.java:477)
        at com.atlauncher.utils.Utils.getMaximumRam(Utils.java:506)
        at com.atlauncher.App.main(App.java:311)
```

### Additional Information
The output of `java -version`:
```
openjdk version "11.0.3" 2019-04-16
OpenJDK Runtime Environment (build 11.0.3+4)
OpenJDK 64-Bit Server VM (build 11.0.3+4, mixed mode)
```

**Comment by dgelessus** [comment]
ATLauncher is not compatible with Java 9 and above (see #294) and you're using Java 11. You need to install Java 8 and start ATLauncher using Java 8.

I'm not familiar with Arch Linux, so I don't know how it handles multiple Java versions. There's probably a way to change your system's default Java version to Java 8, which would be the easiest solution. Alternatively you can specify the full path to Java 8's `java` executable when starting ATLauncher.

**Comment by RyanTheAllmighty** [comment]
Yup as above. Closing as duplicate of #294

**Comment by orblazer** [comment]
Oh ok thanks, now i have an other problem when i start :
![Screenshot_20190427_191417](https://user-images.githubusercontent.com/1344792/56853006-7be63600-6922-11e9-8690-497ad6d058cc.png)

I have try to force GTK 2 and 3 and this have no effect.

**Comment by dgelessus** [comment]
@orblazer Perhaps the fix from issue #145 will help (probably not though, this seems like a different problem). [This answer on Unix StackExchange](https://unix.stackexchange.com/a/138232) has a fix for a similar problem, that might help here.

I would also suggest asking in the support channel of the ATLauncher Discord server (https://atl.pw/discord), perhaps someone there knows a fix for this. (The GitHub bugtracker is only read by a few people who work on the ATLauncher code, whereas the Discord is used by a larger part of the community, so you'll probably get better help in the Discord than here.)
